src/devtools/client/debugger/src/workers/parser/tests/fixtures/scopes/ts-sample.ts

```diff
@@ -1,41 +0,0 @@
-
```

Comment:
i think these files ere important for tests.

could we exclude them?

Reply:
What tests? Our own tests do not need them (AFAIK). And I don't want to enable the typecheck watcher as long as these files are there because they contain errors and that would train people to ignore errors from the typecheck watcher.

Reply:
These fixtures are for the unit tests  in the parser worker.

If it's hard to exclude the files from the watcher, then remove the fixtures and we'll add them later

Reply:
OK, I've excluded them instead